docs/pages/advanced/species.md

```diff
@@ -2,4 +2,294 @@
+Holds array of Talents granted by the species.
+If a value contains a comma `,` Chargen will understand it as "either or" choice.
+```js
+config.speciesTalents.wolfkin = [
+  "Argumentative",
+  "Lightning Reflexes, Warrior Born"
+];
+```
+Above will always give the `Argumentative` Talent and offer a choice between `Lightning Reflexes` and `Warrior Born`.
+For adding Random Talents, see [below](#speciesrandomtalents)
+
+### `speciesRandomTalents`
+Holds an object of table keys and amount of randomized Talents
+```js
+config.speciesRandomTalents.wolfkin = {
+  talents: 1,
+  "talents-wolfkin": 2,
+};
+```
+The above will draw 1 random Talent from generic `Talents - Character Creation` Rolltable that is shipped with WFRP4e Core module 
+and 2 random Talents from user-defined Rolltable with `talents-wolfkin` key.
+
+### `speciesTalentReplacement`
+Holds object where both keys and values are Talent names. It will offer to replace key Talent with value Talent.
+Typically unused with Species. 
+```js
+config.speciesTalentReplacement.wolfkin = {
+    "Artistic": "Beneath Notice"
+};
+```
+Above will allow this Species to make a choice of replacing `Artistic` Talent (if they rolled it from Random Talents) with 
+the `Beneath Notice` Talent.
+
+### `speciesTraits`
+Hold an array of Traits granted by this Species.
+Typically unused by most playable species
+```js
+config.speciesTraits.wolfkin = [
+    "Arboreal"
+];
+```
+
+### `speciesMovement`
+Holds a number that defines the Species' Movement value.
+```js
+config.speciesMovement.wolfkin = 4;
+```
+
+### `speciesFate`
+Holds a number that defines the Species' starting amount of Fate Points.
+```js
+config.speciesFate.wolfkin = 1;
+```
+
+### `speciesRes`
+Holds a number that defines the Species' starting amount of Resilience Points.
+```js
+config.speciesRes.wolfkin = 2;
+```
+
+### `speciesExtra`
+Holds a number that defines the Species' amount of extra Points to distribute between Fate and Resilience.
+```js
+config.speciesExtra.wolfkin = 2;
+```
+
+### `speciesAge`
+Holds a Formula used to determine the character's age. 
+```js
+config.speciesExtra.wolfkin = "10+5d10";
+```
+
+### `speciesHeight`
+Holds an object with data used to determine the character's height.
+```js
+config.speciesHeight.wolfkin = {
+  die: "1d10",
+  feet: 5,
+  inches: 2
+};
+```
+The above means the random height for Wolfkin is `5’2” + 1d10”`
+
+### `speciesCareerReplacements`
+Holds an object describing what available career replacements there are
+```js
+config.speciesCareerReplacements.wolfkin = {
+  "Flagellant": "Hunter"
+}
+```
+The above will allow Wolfkin to pick `Hunter` career in case they rolled `Flagellant`
+
+{: .highlight}
+> This is not typically used for new Species, as you should provide a dedicated Random Career Table
+> 
+> For existing Species it is also recommended to use the `game.wfrp4e.utility.mergeCareerReplacements` helper method instead of direct definitions to avoid overwriting.
```

Comment:
I feel like this is important enough to have an example

Reply:
Fair, but I don't feel it's good to have in a page about _adding new Species_, perhaps this could be an example in World Scripts page?

Reply:
Why not? It's specific to adding species, isn't it just a one liner?

My highlight is a bit off, I specifically been the merge function

Reply:
Added full example at the end of the file and linked it here